readme.txt

```diff
@@ -102,19 +102,23 @@ Learn more (and get detailed instructions) in our [contribute guide](http://jetp
 
 = 3.6 =
 Release Date: July 6th, 2015
+Release Post: http://jetpack.me/2015/07/06/jetpack-3-6-updated-social-widgets-jetpack-cli-and-more/
 
-* Enhancement: CLI: Add a number of Jetpack CLI improvements: see http://jetpack.me/support/jetpack-cli
+Feature Enhancements:
+* CLI: Add a number of Jetpack CLI improvements: see http://jetpack.me/support/jetpack-cli
+* New Jetpack admin page for connection management
+* New Social Media Icons widget
+* FB Like Box: A visual refresh of the Facebook likebox widget
+* Protect: When your IP is blocked, use a math captcha as a fallback instead of a complete block
+
+Full list of changes:
```

Comment:
/s/full list of/Additional since we're taking the items above out of the list below.